metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_utils.py

```diff
@@ -82,7 +82,7 @@ def create_snowsight_base_url(
         if privatelink:
             url = f"https://app.{account_locator}.{cloud_region_id}.privatelink.snowflakecomputing.com/"
         elif cloud == SNOWFLAKE_DEFAULT_CLOUD:
-            url = f"https://app.snowflake.com/{cloud_region_id}/{account_locator}/"
+            url = f"https://app.snowflake.com/{cloud_region_id}.{SNOWFLAKE_DEFAULT_CLOUD}/{account_locator}/"
```

Comment:
It looks like - there is some peculiar behavior in snowflake when it comes to "aws". Please see - https://docs.snowflake.com/en/user-guide/admin-account-identifier#non-vps-account-locator-formats-by-cloud-platform-and-region

`.aws` is not added in account identifier for some cloud regions in aws - namely - `us-west-2, us-east-1, eu-west-1, eu-central-1, ap-southeast-1, ap-southeast-2` . We would need to not append `.aws` for these regions.

Reply:
thanks, I updated based on this